lib/gds_api.rb

```diff
@@ -138,17 +137,14 @@ def self.publishing_api(options = {})
     )
   end
 
-  # Creates a GdsApi::PublishingApiV2 adapter
+  # Creates a GdsApi::PublishingApi adapter
   #
   # This will set a bearer token if a PUBLISHING_API_BEARER_TOKEN environment
   # variable is set
   #
-  # @return [GdsApi::PublishingApiV2]
+  # @return [GdsApi::PublishingApi]
   def self.publishing_api_v2(options = {})
-    GdsApi::PublishingApiV2.new(
-      Plek.find("publishing-api"),
-      { bearer_token: ENV["PUBLISHING_API_BEARER_TOKEN"] }.merge(options),
-    )
+    self.publishing_api(options)
```

Comment:
I think this should create an instance of `GdsApi::PublishingApiV2` still so that it continues to return the same class.

Reply:
I'll remove this commit and fix up.